tasks/build_tags.py

```diff
@@ -37,6 +37,7 @@
     "kubelet",
     "kubeapiserver",
     "cri",
+    "netcgo",
```

Comment:
I think this tag deserves a small comment since it's not a tag we define ourselves :)